### PR TITLE
Bump version to 1.0.4

### PR DESCRIPTION
**Pull request by andytinkham**
Signed-off-by: Andy Tinkham <andy.tinkham@cyberark.com>

### What does this PR do?
Updates base image to 1.0.4 to reflect Postgres version rollback to 10.16 and UBI update to resolve CVE-2021-33910

#### Change log
- [X] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [X] The changes in this PR do not require tests

#### Documentation
- [X] This PR does not require updating any documentation, or
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.0.4] - 2021-07-22
```
Comment (jtuttle) [inline CHANGELOG.md:9]: You also need to add a line at the bottom of the file to make this `[1.0.4]` link work.

**Review by jtuttle — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (andytinkham) [reply CHANGELOG.md:9]: Got it, and updated. Thanks!

Comment (andytinkham) [reply CHANGELOG.md:9]: Oops - missed the Unreleased link update. One sec.

Comment (andytinkham) [reply CHANGELOG.md:9]: OK, all set now.

**Review by jtuttle — APPROVED** [review]

**Pull request merged**